### DISCUSSION TITLE
listing price change

### DISCUSSION THREAD

**Discussion by SimpsonAlex** [Sellbrite/Sellbrite-API#24] [Q&A]
Is it possible to change the price not for the product itself, but for the Listing? I need to set the price and synchronize it with the marketplaces, there can be one SKU but different prices for different marketplaces (using the markup provided by the system is not suitable).

**Comment by rengler-godaddy** [comment]
Hello - Thanks for reaching out! 

There isn’t a way to update the price on listings via the API. The price on your listings can be managed through either price sync, or using a template. If you have further questions about managing the prices on your listings feel free to reach out to support through chat when logged into your account, or by emailing support@sellbrite.com.
